cmd/dep/status.go

```diff
@@ -585,10 +600,13 @@ func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceMana
 
 		// Use the collected BasicStatus in outputter.
 		for _, proj := range slp {
-			out.BasicLine(bsMap[string(proj.Ident().ProjectRoot)])
+			err := out.BasicLine(bsMap[string(proj.Ident().ProjectRoot)])
+			if err != nil {
+				return false, 0, err
+			}
 		}
 
-		out.BasicFooter()
+		err = out.BasicFooter()
```

Comment:
Let's call this error variable something else and have explicit not nil check and return for it. Because reassigning this variable here makes the previous values (line 577 & 579) never being used. And the following return seems to return `errCount`, which is not related to this outputter error.

Reply:
got it, I'll have it return separately rather than continuing to the end if it errors there. (as it is doing with `BasicHeader()`, `BasicLine()`, etc)